infrastructure/minio/stack.env

```diff
@@ -0,0 +1,3 @@
+MINIO_ROOT_USER=hopps
+MINIO_ROOT_PASSWORD=my_secure_pw
```

Comment:
Ist das mit einem Secret ersetzt im Portainer?

Reply:
Nur mittels einer Umgebungsvariable im Portainer Stack. 
Denn eine env var als Secret ist anschneinend nur einer Umgebung vom Typ Docker Swarm Cluster möglich.
[link zur doku](https://docs.portainer.io/user/docker/secrets/add)

Reply:
Man kann im Portainer selbst Secrets angeben. Hast du das gemacht? Diese werden dann als Env Vars in die Container gegeben.

Reply:
@D4ykoo sehr cool! Kannst du das noch im Wiki dokumentieren, wo der Minio läuft? Vielleicht kann man da auch noch ne hübsche URL hinleiten? minio.hopps.cloud / ....app vielleicht? je nachdem, wie eure Konvention hier ist. 😄

Reply:
@manuelhummler dafür gibts grad noch keine DNS Einträge bzw. Issue :) 

@d135-1r43 Leider ist das in der derzeitigen Umgebung nicht möglich. Das geht laut Dokumentation nur in der Docker Swarm Environment. Allerdings habe ich eine Env Var in Portainer gesetzt und diese dem Container übergeben. Die Env Var an sich wird intern allerdings nicht als Secret gehandhabt weshalb diese im UI als Klartext angezeigt wird und nicht encrypted wird. 
![secrets_portainer](https://github.com/user-attachments/assets/36c17030-bb6f-4499-8ff6-7e01589e8e44)

Reply:
Das ist soweit okay. Hauptsache nicht im Git. Ich würde dann auch das Dummy-Passwort aus der `application.properties` rausnehmen.

Reply:
Ne keine Angst, da passe ich auf.